### ISSUE TITLE
Project state query

### ISSUE THREAD

**Issue by lavyadrian** [OpenTournament/OpenTournament#292]
Is this dead? Are you guys still working on it? Because epic's gone and killed unreal and I just heard about it! I really hope to see this in future please hang in there!

**Comment by wail** [comment]
Not at all. It is being developed, please check the activity feed here on GitHub or come chat on Discord.
